src/docfx/Errors.cs

```diff
@@ -411,7 +411,7 @@ public static Error MonikerOverlapping(IEnumerable<string> overlappingmonikers)
         /// Defined duplicate monikers in moniker definition file.
         /// </summary>
         public static Error MonikerNameConflict(string monikerName)
```

Comment:
For `moniker-name-conflict`, we can direct remove it since we won't meet this case.
But for `invalid-uid-moniker`: we may support it in the future, just keep this check and report as info?

Reply:
yes make sense